README.md

```diff
@@ -212,16 +318,86 @@ By the end of this lesson. You should be able to set up two separate servers tha
 
 * **Your Answer:**
 
+This problem actually happens later, not now.
```

Comment:
Should add to a lower level component such as the users container so the end user will see some the main site along with a loading message while the fetch is in progress.